package.json

```diff
@@ -226,6 +226,7 @@
     "----------------------------------------- static analysis -----------------------------------------": "",
     "lint": "eslint src test --quiet --fix",
     "lint-docs": "markdownlint docs",
+    "test-typings-older": "tsc -b test/old-tsconfig.json",
```

Comment:
```suggestion
    "test-typings-older": "tsc --noEmit --emitDeclarationOnly false && tsc -b test/old-tsconfig.json",
```

Reply:
It will not work because with

```
--noEmit --emitDeclarationOnly
```

build options `-b` will not work.